app/src/main/java/fr/free/nrw/commons/upload/MetadataEntry.java

```diff
@@ -0,0 +1,63 @@
+package fr.free.nrw.commons.upload;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+
+/**
+ * COPYRIGHT (C) 2014-2017 WEN YU (YUWEN_66@YAHOO.COM) ALL RIGHTS RESERVED.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Any modifications to this file must keep this entire header intact.
+ *
+ * Change History - most recent changes go on top of previous changes
+ *
+ * MetadataEntry.java
+ */
+public class MetadataEntry {
```

Comment:
Please add Javadoc explaining what this class does, in what context.

Reply:
@nicolas-raoul This class wasn't present in the open source library but this class created some issues while downloading it as jar and then adding it as library. So i had created the class manually and it added it to the project.

Currently I see that we are not using this class anywhere in the project.

- Should I remove this class?

### Or

- Should I keep this class with a documentation so that it would won't create difficulty if we might need this in the future for the project

Reply:
I suggest not adding this class, if it is not needed.

Reply:
Ok then I'll remove it :+1: